### PR TITLE
Presets post routes

### PR DESCRIPTION
**Pull request by bart-de-jonge**
Post routes (and some additional get routes) for stuff with Benine

### CODE REVIEW

**Comment by Markvanderuit** [comment]
Nothing to comment, looks fine to me. Test fails though.

**Comment by geenen124** [comment]
Nice work!

File: routes/benine.js [hunk 1]
```diff
@@ -46,4 +46,67 @@ router.get("/recallTest", (req, res) => {
         });
 });
 
+router.post("/cameras/:cameraId(\\d+)/set-remote-camera-id", (req, res) => {
```
Comment (martijnjanssen) [inline routes/benine.js:49]: I believe you switched the routes here. Also, could you add comments to the routes? What do they do?

File: routes/benine.js [hunk 2]
```diff
@@ -46,4 +46,67 @@ router.get("/recallTest", (req, res) => {
...
+                res.json({ success: true, message: "Preset id stored successfully!" });
+            } else {
+                res.json({ success: false, message: "That's not a valid camera!" });
+            }
+        });
+    }
+    res.json({ success: false, message: "Please provide a remote camera id!" });
+});
+
+router.post("/cameras/:cameraId(\\d+)/set-preset-id", (req, res) => {
```
Comment (martijnjanssen) [inline routes/benine.js:64]: I don't believe we need to assign a preset to a camera.

File: routes/benine.js [hunk 3]
```diff
@@ -46,4 +46,67 @@ router.get("/recallTest", (req, res) => {
...
+            const camera = manager.data.scriptingProject.cameraList[0].camera[req.params.cameraId];
+            if (camera) {
+                camera.presetId = req.body.presetId;
+                res.json({ success: true, message: "Preset id stored successfully!" });
+            } else {
+                res.json({ success: false, message: "That's not a valid camera!" });
+            }
+        });
+    }
+    res.json({ success: false, message: "Please provide a remote camera id!" });
```
Comment (martijnjanssen) [inline routes/benine.js:61]: This needs to be in an else statement.

**Comment by bart-de-jonge** [comment]
I will process the comments and open another PR.

**Comment by martijn9612** [comment]
:cry:

**Comment by martijnjanssen** [comment]
@geenen124 I am dissapoint. :cry:

**Comment by Markvanderuit** [comment]
@geenen124 too soon, man. Too soon.

**Pull request merged**